docs/user-stories/data-discovery.md

```diff
@@ -59,14 +69,71 @@ Watch the video walkthrough below or read the step-by-step instructions.
 
 ### Searching for Data
 
-Let's start on the Aurelius Atlas landing page. Here, the data scientist can enter search terms like "employee",
-"staff", or "workforce". See the screenshot below.
+Let's start on the Aurelius Atlas landing page. Here, the data scientist
+can enter search terms like _employee_, _staff_, or _workforce_. See the
+screenshot below.
 
 [![Data Discovery](../img/screenshots/data-discovery/business-context.jpg)](../img/screenshots/data-discovery/business-context.jpg)
 
 Here's an explanation of the highlighted elements:
 
-1. **Business Context**: The landing page has a dedicated section to help you get started exploring the business
-    context of the data. You can enter search terms related to your business question to find relevant data sets.
-2. **Info Panel**: Click the question mark icon for an explanation of the business context meta model.
-3. **Search Bar**: Queries made through this search bar will have business context filters pre-applied.
+1. **Business Context**: The landing page has a dedicated section to help
+2. you get started exploring the business
+3. context of the data. You can enter search terms related to your business
+4. question to find relevant data sets.
+5. **Info Panel**: Click the question mark icon for an explanation of the
+6. business context meta model.
+7. **Search Bar**: Queries made through this search bar will have business
+8. context filters pre-applied.
+
+To search for employees, click the search icon and enter the term _employee_.
+This will generate a list of results related to employees.
+
+As a data scientist seeking entities representing employees, use the sidebar to
+drill down and filter the results by data entities. Applying this filter will
+narrow the results to 4 entities.
+
+With this smaller set of results, review each entity individually. Compare the
+entities _Personnel_ and _Internal_ by examining their breadcrumbs. The breadcrumbs
+indicate that _Internal_ is a child entity of _Personnel_, showing a hierarchical
+relationship between these concepts.
+
+[![Data Discovery](../img/screenshots/data-discovery/data-discovery-2.jpg)](../img/screenshots/data-discovery/data-discovery-2.jpg)
+
+We are only interested in permanent employees, therefore select the entity
+_Internal_. That will take us to the details of the entity.
+
+[![Data Discovery](../img/screenshots/data-discovery/data-discovery-3.jpg)](../img/screenshots/data-discovery/data-discovery-3.jpg)
+
+In this overview, we can find the following segments:
+
+1. **Name and Type of the Entity**
+2. **Business Hierarchy**: Shows which domain this entity is related to.
+3. **Description**
+4. **People Responsible for this Data**
+5. **Summary of All the Elements of the Page**
+6. **Button to Navigate Through Each Section Quickly**
```

Comment:
There are too many bold items here, it would be better if we only bold the words we really want to highlight. Right now it's just adding noise. Maybe it would help if we add a short description for each point?